memory-viz/src/memory_model.ts

```diff
@@ -88,7 +88,9 @@ export class MemoryModel {
             "height",
             options.height ? options.height.toString() : "800"
         );
-        this.roughjs_config = options.roughjs_config;
+        this.roughjs_config = options.roughjs_config ?? {
```

Comment:
There should not be a default seed value. Use just `{}` as the default `this.roughjs_config` value.